src/solver/modeler/dataSeries/scenarioGroupRepo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "include/antares/solver/modeler/dataSeries/scenarioGroupRepo.h"
+
+#include <stdexcept>
+
+namespace Antares::Solver::Modeler::DataSeries
+{
+void ScenarioGroupRepository::addPairScenarioRankToGroup(
+  std::string groupId,
+  std::pair<unsigned, unsigned> scenarioToRank)
+{
+    scenarioGroups_[groupId] = {scenarioToRank};
```

Comment:
If a key for `groupId` already exists, it will be erased, right ? Is this intended ?

Reply:
No, it's not, I can also raise a (custom) exception in case we try to erase a scenario group with another.

Reply:
Done